recipes/mrcfile/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "mrcfile" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 689bb3a094500d0025abb0bcb24b75558dde3f68fe659c8d610b15aa6180dbc9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - numpy >=1.11.0,<1.16.0
+    - pip
+    - python
+  run:
+    - numpy >=1.11.0,<1.16.0
+    - python
+
+test:
+  imports:
+    - mrcfile
+
+about:
+  home: https://github.com/ccpem/mrcfile
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: MRC file I/O library
+  description: |
+    mrcfile is a Python implementation of the MRC2014 file format
+    (http://www.ccpem.ac.uk/mrc_format/mrc2014.php), which is used in
+    structural biology to store image and volume data.
+
+    It allows MRC files to be created and opened easily using a very
+    simple API, which exposes the file's header and data as numpy
+    arrays. The code runs in Python 2 and 3 and is fully unit-tested.
+
+    This library aims to allow users and developers to read and write
+    standard-compliant MRC files in Python as easily as possible, and
+    with no dependencies on any compiled libraries except numpy. You can
+    use it interactively to inspect files, correct headers and so on, or
+    in scripts and larger software packages to provide basic MRC file
+    I/O functions.
+  doc_url: https://mrcfile.readthedocs.io
+  dev_url: https://github.com/ccpem/mrcfile
+
+extra:
+  recipe-maintainers:
+    - colinpalmer
```

Comment:
@colinpalmer are you OK being a maintainer for this feedstock?

Reply:
Yes, that's fine with me.